content/docs/latest/deploying/registering.md

```diff
@@ -21,7 +21,7 @@ The server will send to the agent a list of all registration entries for workloa
 
 During workload attestation, the agent discovers selectors and compares them to those in the cached registration entries to determine which SVIDs they should assign to the workload.  
 
-You register a workload either by issuing the `spire-server entry create` command at the command line or calling directly into the Registration API, as described in the [Registration API documentation](https://github.com/spiffe/spire/blob/{{< spire-latest "tag" >}}/proto/spire/api/registration/registration.proto). Existing entries can be modified using the `spire-server entry update` command.
+You register a workload either by issuing the `spire-server entry create` command at the command line or calling directly into the Entry API, as described in the [Entry API documentation](https://github.com/spiffe/spire/blob/{{< spire-latest "tag" >}}/proto/spire/api/server/entry/v1/entry.proto). Existing entries can be modified using the `spire-server entry update` command.
```

Comment:
I think that the "registration api" was removed in spiffe/spire#2487, and replaced by the entry api? (@azdagron)

Reply:
It seems like the protos have been moved to their own repo now, the Entry API is [here](https://github.com/spiffe/spire-api-sdk/blob/main/proto/spire/api/server/entry/v1/entry.proto). Would you mind confirming this @azdagron?

Reply:
Yes, that is correct, @mchurichi.